features/post/post-comment/post-comment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from 'react';
+
+import useRelativeTime from '@/shared/hooks/useRelativeTime';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, Typography } from '@/shared/ui';
+import clsx from 'clsx';
+
+import s from './post-comment.module.scss';
+
+export type PostCommentT = {
```

Comment:
The form of writing a type alias with a "T" at the end is not common. Sometimes it is inserted at the beginning of a word when declaring generic type parameters. But even there it is most often redundant and rather a bad habit from users of other programming languages, such as Java, C#.
Remove the trailing "T".